app/controllers/plugin_controller.rb

```diff
@@ -1,58 +1,57 @@
 class PluginController < ApplicationController
-	def user
-	  buffer = []
-	  out = []
+  def user
+    buffer = []
+    out = []
+    if ban = Ban.first(:conditions => ["expiry > UTC_TIMESTAMP() AND steamid = ? AND ban_type = ?", params[:id], Ban::TYPE_SERVER])
+      out << "#USER#"
+      out << "BANNED"
+      out << ban.expiry.utc.to_i
+      out << ban.reason
+      out << "\r\r\r\r\r\r\r"
+    elsif user = User.first(:conditions => {:steamid => params[:id]})
+      teamer = (user.team ? user.teamers.active.of_team(user.team).first : nil)
+      icon = 0
+      rank = "User"
+      if Group.find(Group::DONORS).users.exists?(user)
+        rank = "Donor"
+        icon = icon | 1
+      end
+      if Group.find(Group::CHAMPIONS).users.exists?(user)
+        icon = icon | 2
+      end
+      if user.ref?
+        rank = "Referee"
+        icon = icon | 4
+      end
+      if user.admin?
+        rank = "Admin"
+        icon = icon | 8
+      end
 
-	  if ban = Ban.first(:conditions => ["expiry > UTC_TIMESTAMP() AND steamid = ? AND ban_type = ?", params[:id], Ban::TYPE_SERVER])
-	    out << "#USER#"
-	    out << "BANNED"
-	    out << ban.expiry.utc.to_i
-	    out << ban.reason
-	    out << "\r\r\r\r\r\r\r"
-	  elsif user = User.first(:conditions => {:steamid => params[:id]})
-	    teamer = (user.team ? user.teamers.active.of_team(user.team).first : nil)
-	    icon = 0
-	    rank = "User"
-	    if Group.find(Group::DONORS).users.exists?(user)
-	      rank = "Donor"
-	      icon = icon | 1
-	    end
-	    if Group.find(Group::CHAMPIONS).users.exists?(user)
-	      icon = icon | 2
-	    end
-	    if user.ref?
-	      rank = "Referee"
-	      icon = icon | 4
-	    end
-	    if user.admin?
-	      rank = "Admin"
-	      icon = icon | 8
-	    end
+      buffer << user.steamid
+      buffer << user.username
+      buffer << '0.0.0.0'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.